jetty-core/jetty-compression/jetty-compression-server/src/main/config/etc/jetty-compression-handler.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://jetty.org/configure_10_0.dtd">
+
+<!-- =============================================================== -->
+<!-- Mixin the Compression Handler                                   -->
+<!-- This enabled compression support for the entire server.         -->
+<!-- Compression algorithm support are found in different modules.    -->
+<!-- =============================================================== -->
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
+  <Call name="insertHandler">
+    <Arg>
+      <New id="CompressionHandler" class="org.eclipse.jetty.compression.server.CompressionHandler">
```

Comment:
The `CompressionHandler` seems to completely replace the existing `GZIPHandler`. Should we deprecate the latter then?

Reply:
I don't think we can deprecate GZIPHandler whilst this is experimental.  Once we are happy it is a valid alternative, we can deprecate GZIPHandler